src/__tests__/vueComponent.test.js

```diff
@@ -33,17 +49,124 @@ describe('vue-component', () => {
     })
 
     provide.value('HelloComponent', HelloComponent)
+    provide.value('PersonsComponent', PersonsComponent)
   })
 
   it('should render a vue component with name', () => {
     const elem = compiledElement('<vue-component name="HelloComponent" />')
     expect(elem[0].innerHTML.replace(/\s/g, '')).toBe('<span>Hello</span>')
   })
 
-  it('should render a vue component with properties from scope', inject(($rootScope) => {
-    const scope = $rootScope.$new()
+  it('should render a vue component with vprops object from scope', () => {
+    const scope = rootScope.$new()
     scope.person = { firstName: 'John', lastName: 'Doe' }
     const elem = compiledElement('<vue-component name="HelloComponent" vprops="person" />', scope)
     expect(elem[0].innerHTML).toBe('<span>Hello John Doe</span>')
-  }))
+  })
+
+  it('should re-render the vue component when vprops value changes', (done) => {
+    const scope = rootScope.$new()
+    scope.person = { firstName: 'John', lastName: 'Doe' }
+    const elem = compiledElement('<vue-component name="HelloComponent" vprops="person" />', scope)
+    expect(elem[0].innerHTML).toBe('<span>Hello John Doe</span>')
```

Comment:
These kinds of `expect` look unnecessary because the expected behaviors are *the component should be re-rendered by the change of `vprops` value*. It's better to **have only one logical assertion per test** 😄